### PR TITLE
Splitted strategy button

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
image

https://linear.app/unleash/issue/1-1038/strategy-creation-split-into-two-buttons

### CODE REVIEW

File: frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx [hunk 1]
```diff
@@ -35,25 +50,51 @@ export const FeatureStrategyMenu = ({
...
+                onClick={() => navigate(createStrategyPath)}
                 aria-labelledby={popoverId}
                 variant={variant}
+                size={size}
                 sx={{ minWidth: matchWidth ? '282px' : 'auto' }}
             >
                 {label}
             </PermissionButton>
+
+            <StyledAdditionalMenuButton
```
Comment (sjaanus) [inline frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx:76]: This should be PermissionIconButton

Comment (sjaanus) [reply frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx:76]: Also should have locking behaviour like this

image

Comment (Tymek) [reply frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx:77]: Nope. PermissionIconButton doesn't have outline. https://mui.com/material-ui/react-button/#icon-button

File: frontend/src/component/feature/FeatureStrategy/FeatureStrategyEmpty/FeatureStrategyEmpty.tsx [hunk 2]
```diff
@@ -76,6 +78,9 @@ export const FeatureStrategyEmpty = ({
         onChangeRequestAddStrategyClose,
     } = useChangeRequestAddStrategy(projectId, featureId, 'addStrategy');
 
+    const { uiConfig } = useUiConfig();
+    const strategySplittedButton = uiConfig?.flags?.strategySplittedButton;
```
Comment (sjaanus) [inline frontend/src/component/feature/FeatureStrategy/FeatureStrategyEmpty/FeatureStrategyEmpty.tsx:82]: Useful to add it to server-dev.ts so more people will test it accidently.

**Review by sjaanus — APPROVED** [review]
Small comment, but looks good

**Pull request merged**